packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -94,7 +94,18 @@ export class MetamaskPage implements WalletPage {
       await this.closePopover();
     });
   }
-
+  async setTestnetNetwork(networkName: string) {
```

Comment:
it can be any network
lets rename it to `setNetwork` and remove any testnet mentions